src/dsp_tools/xmllib/models/user_enums.py

```diff
@@ -0,0 +1,8 @@
+from enum import Enum
+from enum import auto
+
+
+class NewlineReplacementTags(Enum):
+    none = auto()
+    paragraph = auto()
+    newline = auto()
```

Comment:
Some suggestions:

- Naming: instead of `NewlineReplacementTags`, how about just `NewlineReplacement`?
- Naming: by convention, variants of an enum are - like global constants - normally all-caps, so the conventional thing to expect would be `NewlineReplacement.NONE`, `NewlineReplacement.PARAGRAPH`, `NewlineReplacement.NEWLINE`
- Naming: The HTML tag `<br/>` is typically "translated" to `linebreak` not `newline`
- Enum: I think this would be a case for using the enum subclass `StrEnum`, but I'm not 100% sure, and it probably doesn't make a difference anyways.